isis/src/viking/apps/vikcal/vikcal.cpp

```diff
@@ -0,0 +1,141 @@
+/** This is free and unencumbered software released into the public domain.
+
+The authors of ISIS do not claim copyright on the contents of this file.
+For more details about the LICENSE terms and the AUTHORS, you will
+find files of those names at the top level of this repository. **/
+
+/* SPDX-License-Identifier: CC0-1.0 */
+
+#include "SpecialPixel.h"
+#include "ProcessByLine.h"
+#include "UserInterface.h"
+#include "Pvl.h"
+#include "CalParameters.h"
+
+#include "vikcal.h"
+
+using namespace std;
+
+namespace Isis {
+  
+  void cal(vector<Buffer *> &in,
```

Comment:
Might make this something more obviously viking specific since it's just in the Isis namespace, and seems like a potentially common function name?

Reply:
I think it's better just to make it static so it can't referenced outside of this file. I think "cal" gives the impression that is does the actual cal work (which is true I think?) compared to having a function that is a variation of vikcal